src/directLine.ts

```diff
@@ -411,11 +413,16 @@ export class DirectLine implements IBotConnection {
             if (options.token && options.conversationId) {
                 this.streamUrl = options.streamUrl;
             } else {
-                console.warn('streamUrl was ignored: you need to provide a token and a conversationid');
+                console.warn('DirectLineJS: streamUrl was ignored: you need to provide a token and a conversationid');
             }
         }
+        
+        const interval = Math.min(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND);
 
-        if (options.pollingInterval !== undefined) {
+        if (options.pollingInterval &&
+            Math.min(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND) < POLLING_INTERVAL_LOWER_BOUND) {
+            console.warn('DirectLineJS: provided pollingInterval is under lower bound (200ms), using default of 1000ms');
```

Comment:
It might be nice to include the provided polling interval here, because if it is `NaN` this message might be confusing.

Reply:
done